attest/core/src/types/report_body.rs

```diff
@@ -235,22 +235,16 @@ impl Ord for ReportBody {
                                 Ordering::Equal => {
                                     match self.0.misc_select.cmp(&other.0.misc_select) {
                                         Ordering::Equal => {
-                                            match self.config_id().cmp(&other.config_id()) {
-                                                Ordering::Equal => {
-                                                    match self.0.config_svn.cmp(&other.0.config_svn)
-                                                    {
-                                                        Ordering::Equal => match self
-                                                            .cpu_security_version()
-                                                            .cmp(&other.cpu_security_version())
-                                                        {
-                                                            Ordering::Equal => self
-                                                                .report_data()
-                                                                .cmp(&other.report_data()),
-                                                            ordering => ordering,
-                                                        },
-                                                        ordering => ordering,
+                                            match self.0.config_svn.cmp(&other.0.config_svn) {
```

Comment:
Added task to #3437 to capture this may need to be brought over if `Ord` needs to persist.